server/src/main/java/com/linecorp/centraldogma/server/mirror/MirrorUtil.java

```diff
@@ -59,15 +61,15 @@ public static String normalizePath(String path) {
      *
      * <p>e.g. git+ssh://foo.com/bar.git/some-path#master is split into:
      * - remoteRepoUri: git+ssh://foo.com/bar.git
-     * - remotePath:    /some-path/
+     * - remotePath:    /some-path
      * - remoteBranch:  master
      *
      * <p>e.g. dogma://foo.com/bar/qux.dogma is split into:
      * - remoteRepoUri: dogma://foo.com/bar/qux.dogma
      * - remotePath:    / (default)
      * - remoteBranch:  {@code defaultBranch}
      */
-    static String[] split(URI remoteUri, String suffix) {
+    public static String[] splitRemoteUri(URI remoteUri, String suffix, @Nullable String defaultBranch) {
```

Comment:
Like mentioned in another comment, I'm unsure why this change is necessary 😅  Would you mind explaining/updating in the PR description?

```suggestion
    public static String[] splitRemoteUri(URI remoteUri, String suffix) {
```